Protection Profile/BiocPP.adoc

```diff
@@ -455,38 +455,12 @@ Target error rates defined in SFR shall be evaluated based on <<BIOSD>>. Normall
 
 ===== FIA_PAD_EXT.1 Presentation attack support [[FIA_PAD_EXT.1]]
 
-*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _biometric modality_] with [*selection*: _enrolment detection_] detection at biometric enrolment and [*selection*: _verification detection equal to or greater than enrolment detection_] detection at biometric verification.
+*FIA_PAD_EXT.1.1*:: The TSF shall provide the [*selection*: _eye, face, fingerprint, vein_] with [*selection*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBE_EXT.3 during biometric enrolment and [*selection, equal or greater than enrolment*: _PAD-L3, PAD-L2, PAD-L1, no PAD_] detection as specified in FIA_MBV_EXT.3 during biometric verification.
 
-*Application Note {counter:remark_count}*:: The <<PAD_Levels>> table provides acceptable options for each modality by row for each selection within the SFR. Selecting a specific modality then specifies the options that are available for enrolment and verification detection.
+*Application Note {counter:remark_count}*:: Not all combinations of modality and PAD levels may be supported as PAD-L2 and PAD-L3 toolboxes are not publicly released.
```

Comment:
I understand why this note is here and the new selection mechanism makes it more concise (details in [this comment](https://github.com/biometricITC/cPP-biometrics/pull/419#issuecomment-2433382642)). Two questions though:

 * We're not hiding the _existence_ of higher PAD testing levels, correct? The way the note is written might imply that we are.
 * Does CC allow for instructions/normative text in a note? To me this is essentially saying you need to go look up the existence of another document before you make this selection, in order to ensure that the selection value is valid. Having the table, while cumbersome, prevented an invalid choice.

Reply:
So honestly I wasn't exactly sure how much we wanted to state exactly which PAD-L2 toolboxes we have. I had sort of left it more "you need to ask". 

I will adjust though to incorporate this (and update the web page as well).